CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.1.4
+
+## Bug fixes
+
+* Added row validation check (#4540).
```

Comment:
Perhaps rephrase this so it makes more sense from a public API standpoint. Not entirely sure how to though, since people could be affected by this from a number of places?